qlib/FixedLengthUtil.qm

```diff
@@ -741,8 +745,13 @@ public class FixedLengthUtil::FixedLengthAbstractWriter {
             if (spacingsize) {
                 string spacing = strmul(fh.padding_char ?? " ", spacingsize);
 
-                if (fh.padding == "left" || (!fh.padding && fh.type == "int"))
```

Comment:
you have made an additional change here - the default documented behavior is to right justify "int" fields and left justify all others.   

I think it makes sense to handle int fields the same as float and number regarding this, but it's not OK to change the padding behavior for int.